### PR TITLE
Fixed issue: 'The team you specified does not exist' (remove command)

### PR DESCRIPTION
**Pull request by selfeki**
#70

### CODE REVIEW

File: src/actions/remove/remove.factory.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 module.exports = (eventExists, mapUsernameToUserInfo, REMOVE) => {
   return (parsedCommand, usernameToIds, myUserId, events) => {
-    const { event, usernames } = parsedCommand;
-    eventExists(event, events);
+    const { from, usernames } = parsedCommand;
+    eventExists(from, events);
     return {
       type: REMOVE,
-      event,
+      from,
```
Comment (grazietta) [inline src/actions/remove/remove.factory.js]: this needs to be changed to event:from instead of from:from

File: src/reducers/index.js [hunk 2]
```diff
@@ -89,15 +89,16 @@ function processUserIds(prevMembers, action) {
...
+        else {
           return member;
         }
       });
     case STOP:
       return prevMembers.map(member => {
-        return member.user_id === action.userInfo.user_id
-          ? Object.assign({}, member, { ignore: true })
-          : member;
+        return member.user_id === action.userInfo.user_id ?
```
Comment (alireza-a) [inline src/reducers/index.js]: you can use an if statement instead of ternary operator

File: src/reducers/index.js [hunk 3]
```diff
@@ -75,11 +75,11 @@ function processUserIds(prevMembers, action) {
       });
     case SKIP:
       return prevMembers.map(member => {
-        return member.user_id === action.userInfo.user_id
-          ? Object.assign({}, member, {
-              skip_until: action.skip_until
-            })
-          : member;
+        return member.user_id === action.userInfo.user_id ?
```
Comment (alireza-a) [inline src/reducers/index.js]: use an if statement instead of ternary operator

File: src/actions/remove/remove.spec.js [hunk 4]
```diff
@@ -13,12 +13,12 @@ describe('remove action', function() {
...
     const parsedCommand = {
       type: REMOVE,
-      event: 'artris',
+      from: 'artris',
       usernames: ['me']
     };
     const expected = {
       type: REMOVE,
-      event: 'artris',
+      from: 'artris',
```
Comment (grazietta) [inline src/actions/remove/remove.spec.js]: You might need to change this 'from' key to 'event' on this line.

**Pull request merged**